src/_guides/bump-sh-tutorials/fastapi.md

```diff
@@ -12,7 +12,8 @@ The following assumes your local machine is configured with Python and FastAPI,
 
 1. [Create and name](https://bump.sh/docs/new?utm_source=bump&utm_medium=content_hub&utm_campaign=getting_started) your first API documentation. Then, retrieve the name and token of this documentation from the _CI deployment_ settings page.
 
-2. Install the Bump.sh CLI with [npm](https://docs.npmjs.com/cli/v9/configuring-npm/install?v=true) as below, or use [alternative options](/help/continuous-integration/cli), with
+2. Install the Bump.sh CLI with [npm](https://docs.npmjs.com/cli/v9/configuring-npm/install?v=true) as below, or use [alternative options](/help/continuous-integration/cli).
```

Comment:
The 3. and the 4. are rendered as 1. and 1.
Same same in all guides when previous paragraph finish by a bash. 
I don't remember the tip to solve this. Do you have it @paulRbr ?

Reply:
@fbraure I've reverted changes to other guides and will fix them as I work on them to keep collateral damage to a minimum.